.github/CONTRIBUTING.md

```diff
@@ -10,30 +10,52 @@ Be sure to follow our [code of conduct](https://github.com/ScratchAddons/Scratch
 
 Found a bug? Have an idea? You can open an [issue](https://github.com/ScratchAddons/ScratchAddons/issues) to report bugs and send feedback! But first, be sure to [search for existing issues](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
```

Comment:
I don't think the second link is necessary since it just links to the main issues page again.

Reply:
I have updated the first link - now "open an issue" links to the issue creation page, and the second one still links to the listing page.